src/utils/aotf.js

```diff
@@ -432,9 +432,30 @@ export function processMutations (mutations, types) {
     mutation._icon = mutationIcons[mutation.name] || mutationIcons['']
     mutation._shortDescription = getMutationShortDesc(mutation.description)
     mutation._help = getMutationExtendedDesc(mutation.description)
+    mutation._validStates = getStates(mutation.description)
     processArguments(mutation, types)
   }
 }
+/**
+ * Get the workflow states that the mutation is valid for.
+ *
+ * @export
+ * @param {string=} text - Full mutation description.
+ * @return {Array<String>}
+ */
+export function getStates (text) {
```

Comment:
Would be good to unit test this something like:

```js
it('gets valid states', () => {
  expect(get_states('foo\nValid states: running')).to.deep.equal(['running'])
})
```

It would also be good to add a function test for this. To do so edit [this file](https://github.com/cylc/cylc-ui/blob/master/src/services/mock/json/IntrospectionQuery.json) to add a few `Valid for: ...` lines into the descriptions. Then add a test to [this file](https://github.com/cylc/cylc-ui/blob/master/tests/e2e/specs/aotf.cy.js) to make sure that mutations which aren't valid for a running workflow are filtered out (I think the "one" workflow is running).